### PR TITLE
Some mappings for falling blocks, food items and ParticleArgumentType

### PR DESCRIPTION
**Pull request by Juuxel**
(I have no idea why those previous commits are there...)

### CODE REVIEW

File: mappings/net/minecraft/block/AnvilBlock.mapping [hunk 1]
```diff
@@ -11,8 +11,15 @@ CLASS bfz net/minecraft/block/AnvilBlock
 	METHOD <init> (Lbgs$c;)V
 		ARG 1 settings
 		ARG 2 stateBuilder
+	METHOD a configureEntity (Laox;)V
```
Comment (SquidDev) [inline mappings/net/minecraft/block/AnvilBlock.mapping]: I wonder if something like `configureFallingBlock` would be better? `Entity` is a little vague, though I realise it's obvious once you see the method signature.

Comment (Juuxel) [reply mappings/net/minecraft/block/AnvilBlock.mapping]: `configureFallingBlock` could be confusing as the class itself is called FallingBlock.

Comment (shadowfacts) [reply mappings/net/minecraft/block/AnvilBlock.mapping]: Maybe not `configureFallingBlock`, but `configureEntity` is definitely too vague.

File: mappings/net/minecraft/block/AnvilBlock.mapping [hunk 2]
```diff
@@ -11,8 +11,15 @@ CLASS bfz net/minecraft/block/AnvilBlock
...
+		ARG 1 entity
 	METHOD a getPlacementState (Laus;)Lbpm;
 		ARG 1 ctx
+	METHOD a (Lbbp;Let;)V
+		ARG 2 pos
+	METHOD a (Lbbp;Let;Lbpm;Lbpm;)V
+		ARG 2 pos
+		ARG 3 state
```
Comment (shadowfacts) [inline mappings/net/minecraft/block/AnvilBlock.mapping]: There are two state parameters here, what's the difference? The names should reflect that

**Review by shadowfacts — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/AnvilBlock.mapping (hunk 2 above)
Comment (Juuxel) [reply mappings/net/minecraft/block/AnvilBlock.mapping]: I've accidentally mapped that state parameter, it's supposed to be unmapped for this PR.

File: mappings/net/minecraft/block/AnvilBlock.mapping (hunk 1 above)
Comment (Juuxel) [reply mappings/net/minecraft/block/AnvilBlock.mapping]: Maybe `configureFallingBlockEntity`?

**Comment by asiekierka** [comment]
If nothing changes, I'm going to close this pull request by next snapshot due to inactivity.

File: mappings/net/minecraft/block/AnvilBlock.mapping (hunk 2 above)
Comment (Juuxel) [reply mappings/net/minecraft/block/AnvilBlock.mapping]: Fixed.

File: mappings/net/minecraft/block/AnvilBlock.mapping (hunk 1 above)
Comment (Juuxel) [reply mappings/net/minecraft/block/AnvilBlock.mapping]: I've changed it to `configureFallingBlockEntity`.

**Comment by Juuxel** [comment]
@shadowfacts Could you review again?

**Comment by 0f-0b** [comment]
> (I have no idea why those previous commits are there...)

Create a [branch](https://git-scm.com/docs/git-branch) for every PR :)

**Comment by Juuxel** [comment]
> Create a branch for every PR :)

Yeah, I really should.

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**